S/SBML/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env julia
+using BinaryBuilder, Pkg
+
+julia_version = v"1.5.3"
+
+name = "SBML"
+version = v"5.19.0"
+sources = [
+        ArchiveSource(
+          "https://github.com/sbmlteam/libsbml/archive/v5.19.0.tar.gz",
+          "127a44cc8352f998943bb0b91aaf4961604662541b701c993e0efd9bece5dfa8"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir
+SBMLNAME=libsbml-*
+
+cd ${WORKSPACE}/srcdir/${SBMLNAME}
+mkdir build
+cd build
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${prefix} \
+  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+  -DCMAKE_LIBRARY_PATH=${prefix}/lib \
+  -DCMAKE_INCLUDE_PATH="${prefix}/include/libxml2;${prefix}/include" \
+  ..
+make -j${nproc}
+make install
+
+LICENSEDIR=${prefix}/share/licenses/SBML
+mkdir -p $LICENSEDIR
+cp ${WORKSPACE}/srcdir/${SBMLNAME}/LICENSE.txt $LICENSEDIR/LICENSE
+cp ${WORKSPACE}/srcdir/${SBMLNAME}/COPYING.txt $LICENSEDIR/COPYING
+"""
+
+platforms = expand_cxxstring_abis(supported_platforms())
+
+products = [
+    LibraryProduct("libsbml", :libsbml),
+]
+
+dependencies = [
+    Dependency("CompilerSupportLibraries_jll"),
+    Dependency("XML2_jll"),
+    Dependency("Zlib_jll"),
+]
+
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+    preferred_gcc_version=v"10",
```

Comment:
Also, using GCC 10 isn't a good idea, the generated tarball may not work with any Julia version (depending on whether there are versioned GLIBCXX symbols in the resulting tarball)

Reply:
Which is _exactly_ [what happened](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=8665&view=logs&j=fc0f56f8-a80b-5c9f-f500-5d3109a216c4&t=b2285613-6a2a-5fc3-0f53-eccf7552d966&l=15800):
```
ERROR: could not load library "/agent/_work/1/s/S/SBML/build/x86_64-linux-gnu-cxx11/6ala270l/destdir/lib/libsbml.so.5.19.0"
/usr/local/bin/../lib/julia/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /agent/_work/1/s/S/SBML/build/x86_64-linux-gnu-cxx11/6ala270l/destdir/lib/libsbml.so.5.19.0)
┌ Warning: lib/libsbml.so.5.19.0 cannot be dlopen()'ed
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/vDNXJ/src/Auditor.jl:163
```
:slightly_smiling_face:

Reply:
Ah okay, thanks a lot for explanation. I was forcing a higher c++ version because of some c++17 features that I thought were needed, I'll try whether it's really necessary. Thanks for advice.

Reply:
I wouldn't go beyond GCC 7 if possible, because that's what the version of libstdc++ used by the official binaries.

Reply:
OK, the requirement is removed, seems to work locally, let's see what CI says.